integration/auth/auth_test.go

```diff
@@ -235,3 +223,7 @@ func postRequest(url string, req []byte) ([]byte, error) {
 	}
 	return ioutil.ReadAll(resp.Body)
 }
+
+func deleteUser(uid string) {
+	client.DeleteUser(context.Background(), uid)
```

Comment:
This is intended to be a best effort delete operation. I don't think we want to fail a test because the clean up operation failed. Added a comment stating this.

Reply:
I still think it would be good to print out a warning when it fails.